mri/tests/test_fourier_adjoint.py

```diff
@@ -166,18 +166,24 @@ def test_adjoint_stack_3D(self):
         for channel in self.num_channels:
             print("Testing with num_channels=" + str(channel))
             for i in range(self.max_iter):
-                _mask = np.random.randint(2, size=(self.N, self.N))
-                _mask3D = np.asarray([_mask for i in np.arange(self.N)])
-                _samples = convert_mask_to_locations(_mask3D.swapaxes(0, 2))
+                mask = np.random.randint(2, size=(self.N, self.N))
+                sampling_z = np.random.randint(2, size=self.N)
+                sampling_z[self.N//2-10:self.N//2+10] = 1
+                Nz = sampling_z.sum()
+                mask = convert_mask_to_locations(mask)
+                z_locations = np.repeat(convert_mask_to_locations(sampling_z),
```

Comment:
Let's try to avoid this format for multi-line: it's difficult to read on a small screen (or github split diff), and it creates future unnecessary changes.

Reply:
on it. I didnt happen to see that its this terrible